### ISSUE TITLE
setCustomAutocompletion does not work with keepCurrentCompleters=true

### ISSUE THREAD

**Issue by rappenze** [F0rce/ace#36]
**Describe the bug**
I tried to add more than one custom auto completion category by calling the method several times. But only the last call is transfered to the client side.

**To Reproduce**
Add the following code:
```
AceEditor aceEditor = new AceEditor();
aceEditor.setMode(AceMode.sql);
aceEditor.setCustomAutocompletion(Arrays.asList("User", "Role"), "table", true);
aceEditor.setCustomAutocompletion(Arrays.asList("FullName", "Address", "Right"), "column", true);
aceEditor.setAutoComplete(true);
```

**Expected behavior**
I expect that auto complete shows all values, but only values with category "column" are shown.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome
 - Version: 3.3.3

**Additional context**
Vaadin version 23.1.

**Comment by F0rce** [comment]
@rappenze thanks for the issue

I was able to reproduce it and found why and where it happens. I'll try to build a hotfix asap.

**Comment by F0rce** [comment]
Fixed with v3.4.0 (https://vaadin.com/directory/component/ace/3.4.0)

-- Backport to Vaadin 14 soon
